docs/notes.rst

```diff
@@ -16,8 +21,12 @@ It also adds the following bug fixes and improvements:
 * Fix for setting *nullptr* in a *fixed_string*
 * Fix for v1.8.1 not building with -DBUILD_JAVA=ON
 * Fix for GAP messages not being sent in some cases
+* Fix for local/remote locators management
```

Comment:
I would not mention this, as it was introduced in 1.9.0-beta and was not present in 1.8.1